src/main/java/com/minecolonies/coremod/colony/buildings/buildingviews/package-info.java

```diff
@@ -0,0 +1,4 @@
+/**
+ * Contains all the view for the buildings
+ */
+package com.minecolonies.coremod.colony.building.buildingviews;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This file "package-info.java" should be located in "com/minecolonies/coremod/colony/building/buildingviews" directory, not in "/home/travis/build/Minecolonies/minecolonies/src/main/java/com/minecolonies/coremod/colony/buildings/buildingviews". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1598)